public/resources/ts/common-defer.ts

```diff
@@ -143,9 +142,7 @@ function setCheckedTheme(theme: string) {
 
 setCheckedTheme(localStorage.getItem("currentTheme") ?? "default");
 
-tippy("*[data-tippy-content]", {
-  boundary: "window",
```

Comment:
any clue why this was using `boundary: window` instead of the default `scrollParent`?

Reply:
I remember it was me that added it and it fixed a bug that we had been trying to fix for a long time but I can't remember the details

Reply:
here is where I did it https://github.com/SkyCryptWebsite/SkyCrypt/commit/eac7a231b62d5c9e9b080b2d444eb2160a8fe531

Reply:
it seams to be fixing #204

Reply:
after some testing this PR does not appear to reintroduce #204